crypto/asn1/a_gentm.c

```diff
@@ -25,6 +27,39 @@ int asn1_generalizedtime_to_tm(struct tm *tm, const ASN1_GENERALIZEDTIME *d)
     return asn1_time_to_tm(tm, d);
  }
 
+/* Inverse of asn1_generalizedtime_to_tm ()*/
+ASN1_GENERALIZEDTIME *asn1_generalizedtime_from_tm(ASN1_GENERALIZEDTIME *s, struct tm *ts)
+{
+    char* p;
```

Comment:
And I have changed my mind, after looking at the code a bit more... update coming.